### PR TITLE
fix(CT-5002): Line items disregarded on expenses exported in a currency different than the payment one

### PR DESCRIPTION
**Pull request by SvetlinStaevPayhawk**
Excluded currency check from the line items check

### CODE REVIEW

**Review by alexanderfilipov — APPROVED** [review]

File: src/managers/integration/Manager.spec.ts [hunk 1]
```diff
@@ -813,11 +813,23 @@ describe('integrations/Manager', () => {
                         },
                     ],
                     lineItems: [{
-                        id: 'test',
+                        id: 'test-1',
                         reconciliation: {
                             ...reconciliation,
-                            expenseTotalAmount: 10,
-                            accountCode: 'line_item',
+                            expenseTotalAmount: 7,
```
Comment (vassildinev) [inline src/managers/integration/Manager.spec.ts:819]: how does this test verify negative amount on line item?

File: src/managers/integration/Manager.ts [hunk 2]
```diff
@@ -548,21 +548,20 @@ export class Manager implements IManager {
...
+
+        // In case of different currencies convert the tax amount to payment currency
+        if (expenseCurrency !== paymentCurrency && taxAmount && expenseCurrency) {
+            fxRate = await this.fxRates.getByDate(expenseCurrency, paymentCurrency, expenseDate);
+        }
 
+        if (!expense.lineItems || expense.lineItems.length === 0) {
             const trackingCategories = this.extractTrackingCategories(expense.reconciliation.customFields2, logger);
             const lineItem: XeroEntities.ILineItem = {
                 amount: Math.abs(expenseTotalAmount),
```
Comment (vassildinev) [inline src/managers/integration/Manager.ts:563]: is this amount converted into payment currency?

File: src/managers/integration/Manager.spec.ts [hunk 3]
```diff
@@ -871,10 +883,15 @@ describe('integrations/Manager', () => {
                         files,
                         url: `${portalUrl}/expenses/${encodeURIComponent(expenseId)}?accountId=${encodeURIComponent(accountId)}`,
                         lineItems: [{
-                            amount: 10,
+                            amount: 7,
                             taxAmount: expense.lineItems![0].reconciliation.expenseTaxAmount,
                             accountCode: expense.lineItems![0].reconciliation.accountCode,
                             taxType: expense.taxRate?.code,
+                        }, {
+                            amount: 3,
```
Comment (vassildinev) [inline src/managers/integration/Manager.spec.ts:891]: not sure what exactly this test verifies, but shouldn't we have a test where the expense is in EUR while the payment is in USD and there are line items?

File: src/managers/integration/Manager.ts (hunk 2 above)
Comment (SvetlinStaevPayhawk) [reply src/managers/integration/Manager.ts:563]: It should be converted as well indeed.

File: src/managers/integration/Manager.spec.ts (hunk 3 above)
Comment (SvetlinStaevPayhawk) [reply src/managers/integration/Manager.spec.ts:891]: This test is exactly like that, but had only a single line item even as it mentioned `creates bill with payments and multiple line items when expense has settled transactions`, this is why I reused it to check the solution.

**Review by vassildinev — APPROVED** [review]

**Pull request merged**